### PR TITLE
PW-3785 Added AfterPay support in Hybris

### PR DESCRIPTION
**Pull request by saquibsayyad**
**Description**
Added AfterPay payment method.

**Tested scenarios**
Tested payment flow using AfterPay

### CODE REVIEW

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js [hunk 1]
```diff
@@ -120,6 +121,17 @@ var AdyenCheckoutHybris = (function () {
                 }
             }
 
+            if (paymentMethod === "afterpay_default") {
+                if (!this.afterPay.state.isValid) {
```
Comment (martinsrenato) [inline adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:125]: Not sure if this is already happening, but it's nice to show the fields in red on submit if they're not filled in. You might want to check if it's triggering that behaviour, if not it should work if we call `this.afterPay.showValidation()` before this `isValid` check.
This is a tip given by Marc from checkout when I asked about credit cards, but he said it should work for all components.

Comment (saquibsayyad) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:125]: Right now it only marks red if field was selected but not filled. I will add this.

Comment (saquibsayyad) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:125]: added it.

**Review by martinsrenato — DISMISSED** [review]

**Review by Aleffio — DISMISSED** [review]

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js [hunk 2]
```diff
@@ -427,6 +440,23 @@ var AdyenCheckoutHybris = (function () {
             }
         },
 
+        showAfterPay: function (countryCode) {
```
Comment (rkewlani) [inline adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js:443]: can we please rename this to initiateAfterPay to align the naming with other PMs?

**Review by rkewlani — DISMISSED** [review]

File: adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js (hunk 2 above)
Comment (saquibsayyad) [reply adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js]: done

**Review by rkewlani — APPROVED** [review]

**Review by martinsrenato — APPROVED** [review]

**Pull request merged**